wizard/views/load.py

```diff
@@ -0,0 +1,22 @@
+import streamlit as st
+import mlcroissant as mlc
+import tempfile
+from pathlib import Path
```

Comment:
We preferred using [`epath.Path`](https://github.com/mlcommons/croissant/blob/8d26bdb26b50b3024ed8354eec585822a901bb81/python/mlcroissant/mlcroissant/_src/core/constants.py#L98) in mlcroissant. The API is a bit easier to use and its integration with gcs:// can prove useful in the future. WDYT?